simSPI/crd.py

```diff
@@ -0,0 +1,135 @@
+"""Helper functions for tem.py to write particle data to .crd file."""
+import math
+import os
+
+import numpy as np
+from scipy.stats import special_ortho_group
+
+
+def write_crd_file(
+    numpart,
+    xrange=np.arange(-100, 110, 10),
+    yrange=np.arange(-100, 110, 10),
+    crd_file="crd.txt",
+):
+    """Write particle data to .crd file.
+
+    Parameters
+    ----------
+    numpart : int
+        Number of particles.
+    xrange : ndarray
+        Valid horizontal range for display.
```

Comment:
units of pixels?

Reply:
Sorry, worded this really poorly. It's an array containing all x-coordinates of particle centers to write to the coordinate file.

Reply:
changes reflected in the next push (coming soon)